bin/yb-ctl

```diff
@@ -821,7 +820,8 @@ class ClusterControl:
         self.parent_parser = argparse.ArgumentParser(add_help=False)
         self.setup_parent_parser()
         self.parser = argparse.ArgumentParser()
-        self.subparsers = self.parser.add_subparsers()
+        self.subparsers = self.parser.add_subparsers(dest='command')
+        self.subparsers.required = True
```

Comment:
shouldn't there be some default `help` that it defaults to if you don't specify this?

Reply:
https://stackoverflow.com/questions/22990977/why-does-this-argparse-code-behave-differently-between-python-2-and-3